package.json

```diff
@@ -74,13 +79,26 @@
     "lint": "yarn run lint:css && yarn run lint:js",
     "lint:css": "stylelint src/scss/**/*.scss",
     "lint:js": "eslint --ext .js .",
-    "prepare": "concurrently -n \"lint,compile,test\" -c \"blue,yellow,green\" \"yarn lint\" \"yarn compile\" \"yarn test\" --kill-others-on-fail",
+    "prepare": "concurrently -n \"lint,compile,test,prepare:css\" -c \"blue,yellow,green,#008080\" \"yarn lint\" \"yarn compile\" \"yarn test\" \"yarn prepare:css\" --kill-others-on-fail",
     "test": "concurrently -n \"test:js,test:scss\" -c \"cyan,magenta\" \"yarn test:js\" \"yarn test:scss\"",
     "test:js": "jest --config=jest.config.js",
     "test:scss": "jest --config=src/test/scss/jest.config.js",
     "test:build": "sass --no-source-map --load-path=node_modules --style=compressed src/scss:dist/css",
     "test:cover": "jest --collect-coverage",
-    "test:cover:CI": "cat coverage/lcov.info | coveralls"
+    "test:cover:CI": "cat coverage/lcov.info | coveralls",
```

Comment:
NIT – I'd put these into alphabetical order in the scripts list. So move autoprefixer and build to the top, prepare:css next to prepare etc.